app/helpers/proposals_helper.rb

```diff
@@ -22,6 +22,30 @@ def proposal_type_year(proposal_type)
     proposal_type.year&.split(",")&.map { |year| year }
   end
 
+  def proposal_same_week_as
+    Proposal.all.where(outcome: "approved").map { |pro| [pro.code] }
+  end
+
+  def assigned_dates(location)
+    return "" unless location
+
+    start_date = location.start_date
+    end_date = location.end_date
+    get_assigne_dates(start_date, end_date, location)
```

Comment:
"assign" has no "e" on the end

Also excluding dates is the opposite of assigning them, so this naming is confusing. A better method name would be `get_excluded_dates`.